.gitignore

```diff
@@ -51,7 +51,6 @@ nosetests.xml
 coverage.xml
 *,cover
 .hypothesis/
-.vscode/
```

Comment:
I think you misunderstood?

The `.gitignore` is files or paths that git itself should ignore.
So with `.vscode/` in the `.gitignore` it won't automatically add any files under that path.
Which is what we want.
(The docs about .gitignore: https://git-scm.com/docs/gitignore )

The comment was about the `.vscode/settings.json` file, which should not be committed.
And the entry in the .gitignore should stay.

Reply:
Hi @grigi, yes I misunderstood. I guess I didn't read carefully enough your remarks. Personal settings shouldn't be committed to github. I wasn't aware of the fact, that I did. Which would have been clear if I had read carefully. *loop*! But I'm happy that `vscode/` is an entry in `.gitignore`. I reverted to the previous commit for this file.